bundles/model/org.eclipse.smarthome.model.item/src/org/eclipse/smarthome/model/item/internal/GenericItemProvider.java

```diff
@@ -340,8 +337,6 @@ private void internalDispatchBindings(BindingConfigReader reader, String modelNa
     }
 
     /**
-     * {@inheritDoc}
-     * <p>
```

Comment:
Actually, you are right - there is no JavaDoc in the parent but only in the inherited class.  So, I will update the PR and remove the JavaDoc from here and move it to ModelRepositoryChangeListener.